### DISCUSSION TITLE
MacosX Viewer Error : HTTP ERROR 500 javax.servlet.ServletException: javax.servlet.ServletException: File [&#47;webcontent&#47;birt&#47;pages&#47;layout&#47;FramesetFragment.jsp] not found

### DISCUSSION THREAD

**Discussion by phoarau** [eclipse-birt/birt#935] [General]
Hi guys,

I have just installed Birt 4.9.0 in MACOS Big Sur. 
I am a long time birt user. Need to do some tricks before let it work as usual.
When i test my first report the viewer crash with :
**HTTP ERROR 500 javax.servlet.ServletException: javax.servlet.ServletException: File [&#47;webcontent&#47;birt&#47;pages&#47;layout&#47;FramesetFragment.jsp] not found**

Has someone have the same issue? I don't even know if it is an issue or a misconfiguration.

Thanks for help.
PS: excuse my poor english, i'm french ...

**Comment by claesrosell** [comment]
We have had similar issues reported but we have not been able to reproduce them.

Do you have JBoss Tools installed?

**Reply by phoarau** [reply]
No, I've only install the all_in_one MACOS version. 
Which tools will help?

**Reply by claesrosell** [reply]
I experienced this error myself on my Linux machine and solved it (#943). If you feel brave you can test to install the latest build of master and see if it works for you.
Latest build is available here: https://github.com/eclipse/birt/actions/runs/2225351625
